package.json

```diff
@@ -28,25 +28,24 @@
   "devDependencies": {
     "@babel/core": "^7.18.13",
     "@babel/preset-env": "^7.18.10",
-    "@commitlint/cli": "^17.0.3",
+    "@commitlint/cli": "^17.1.2",
     "@lwc/eslint-plugin-lwc-internal": "link:./scripts/eslint-plugin",
     "@lwc/jest-utils-lwc-internals": "link:./scripts/jest/utils",
     "@rollup/plugin-node-resolve": "^13.3.0",
     "@rollup/plugin-replace": "^4.0.0",
-    "@rollup/plugin-typescript": "^8.3.4",
-    "@types/acorn": "^4.0.6",
```

Comment:
This is a stub package and no longer needed because acorn ships with its own types now.